modyn/config/schema/modyn_base_model.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class ModynBaseModel(BaseModel):
+    class Config:
+        extra = "forbid"
```

Comment:
Sorry, I don't understand yet what this is doing and why we need it.

Reply:
Ok now I see it has been there before but any reason you renamed this (and @robinholzi why does it exist in the first place?)

Reply:
> why it exists
It is created by me in a previous PR, to be default disallow any field from json that is unknown to the schema. https://github.com/eth-easl/modyn/pull/436

> why I rename it
Previously it is called `BaseModel` which is the same name as the one by default provided by Pydantic. To avoid confusion I rename it.

Reply:
I think that's the cleaner solution now, thanks! Forbidding extra fields is useful for detecting misconfigurations on fields with defaults.